cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java

```diff
@@ -275,30 +275,6 @@ public void testUnauthorized() throws Exception {
     assertUnauthorized(new StreamId("ns1", "stream1"), getUser("admin1"), Action.EXECUTE);
   }
 
-  @Test
-  public void testCache() throws Exception {
```

Comment:
This test was just testing the eviction since we were using a custom weight function for eviction. The existing unit tests will test the caches. However, I plan to add performance tests to test the efficiency of the caches.